examples/django_demo/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django
+factory_boy
```

Comment:
It's actually already on the ``PYTHONPATH`` (this way, we test with the local code, not the packaged version).